### ISSUE TITLE
Returns the wrong Syndicated Link

### ISSUE THREAD

**Issue by chrisaldrich** [diplix/OwnYourCheckin#7]
In using this with micropub on WordPress and the https://github.com/dshanske/syndication-links plugin, I notice that the syndicated link that's returned is for the actual Foursquare location rather than the technical Swarm PESOS checkin.

Example: http://boffosocko.com/2016/05/17/55672674/
The first syndicated link under the post is the one autogenerated by the micropub portion (I think) and the second one is the actual "syndicated" version which appears on Swarm (and which I copied in by hand).

Perhaps this is an odd quirk caused by the data returned by OwnYourCheckin as it dovetails with @dshanske https://github.com/dshanske/syndication-links. (I know it polls several locations to autoimport syndicated links, but I'm not sure how, or where it looks).

I notice that OwnYourGram doesn't provide an autogenerated syndication link like this (or at least it never has for me, and I've always copied it by hand), though it would be nice if it did it automatically. Perhaps this issue https://github.com/aaronpk/OwnYourGram/issues/20 on OwnYourGram could be troubleshot using OwnYourCheckin, and vice versa since I'm getting a syndication link with the Checkin version, but not the Gram version.

**Comment by dshanske** [comment]
I can't speak to that. The Micropub plugin stores the data supplied to it. The Syndication Links plugin just saves it.

**Comment by chrisaldrich** [comment]
<a href="https://github.com/diplix/OwnYourCheckin/issues/7" class="u-in-reply-to"></a>I'll also ping this related issue on micropub which may benefit from the apparent difference between the two similar based services either being able to provide a syndicated version or not:
https://github.com/snarfed/wordpress-micropub/issues/21

**Comment by diplix** [comment]
i’m sorry, i can’t quite follow, because what i see is, that both the syndication links in [your example](http://boffosocko.com/2016/05/17/55672674/) are being send via micropub. this is what ownyourgram sent in your case:

```
Array
(
    [mp-type] => checkin
    [published] => 2016-05-17T23:21:40+02:00
    [location] => geo:34.128413917767,-118.24102163315
    [place_name] => Little Free Library #8424
    [place_url] => https://foursquare.com/venue/55ce3e31498ed1924c3ad398?ref=YOFY2SHCUPCRRJUB
    [category] => Array
        (
            [0] => Library
            [1] => checkin
            [2] => foursquare
        )

    [content] => Checking out what's new at Little Free Library #8424.
    [syndication] => https://foursquare.com/forward/checkin/573b8b64498e7896508ab004
    [place_icon_url] => https://ss3.4sqi.net/img/categories_v2/building/library_bg_120.png
)
```

the first syndication link on your page is sent as [place_url] and the second as actual [syndication] link.

i’m not sure how @dshanske ’s syndication-links plugin works or why it doesn’t get the syndication link (doesn’t it?) or what needs to be changed in ownyourgrams micropub request to make wordpress get the micropubed links, but the data seems to be where it’s supposed to be.

**Comment by dshanske** [comment]
Micropub for WordPress 0.4 which was just released to stable saves all passed properties into metadata with the prefix mf2_ and Syndication Links displays any URLs saved in the mf2_syndication property.

**Comment by chrisaldrich** [comment]
I double checked on my side, and it does appear that the `[syndication]=> https://foursquare.com/forward/checkin/573b8b64498e7896508ab004` code did come from OwnYourCheckin, apologies for the confusion -- I'm used to Syndication Links putting the internal data before the manual data. 

Hopefully the differences between the two help to track down the error on https://github.com/snarfed/wordpress-micropub/issues/21 if they weren't fixed with the 0.4v.
